x-pack/plugins/ml/public/alerting/result_type_selector.tsx

```diff
@@ -8,8 +8,7 @@
 import { EuiCard, EuiFlexGroup, EuiFlexItem, EuiFormRow } from '@elastic/eui';
 import { FormattedMessage } from '@kbn/i18n/react';
 import React, { FC, useMemo } from 'react';
-import { ANOMALY_RESULT_TYPE } from '../../common/constants/anomalies';
-import { AnomalyResultType } from '../../common/types/anomalies';
+import { AnomalyResultType, ANOMALY_RESULT_TYPE } from '@kbn/ml-utils';
```

Comment:
do we want to be exporting everything from the root of `ml-utils`?
i can see how it is useful for external plugins who may not know where things live, but for our own includes, I find it useful to know i'm importing from `common/constants/anomalies`

Reply:
Plus I'd be surprised if we don't have some naming clashes

Reply:
Since all the code lives in kibana itself and these packages are not published to `npm` (which might strip everything in `src` and only publish the built file of the package) it should be safe to deep link into `src` like we do into `plugins/ml/common`, for example:

```
import { ANOMALY_RESULT_TYPE } from '@kbn/ml-utils/src/constants/anomalies';
import { AnomalyResultType } from '@kbn/ml-utils/src/types/anomalies';
```

I pushed an updated with that change. I'm still exploring how this change affects our own plugin's bundle size and that of consuming plugins.